omero/sysadmins/windows/server-installation.txt

```diff
@@ -83,11 +62,14 @@ Python 2
 
 **Make sure you install the correct Python 2 version for your Ice version:**
 
-- If you have downloaded Ice 3.5.1 from our downloads page, you will
-  need to install the `latest Python 2.7 release <http://www.python.org/downloads/windows/>`_.
+- If you have downloaded Ice 3.5.1 from our downloads page or Ice
+  3.6.0 from ZeroC (note that support is currently upcoming), you will
```

Comment:
I know you explain 'upcoming' = experimental support in the versions requirements page but incase someone hasn't read that, it wouldn't hurt to have that stated explicitly here too.